doc_build_aliases.sh

```diff
@@ -92,12 +92,18 @@ alias docbldeas='$GIT_HOME/docs/build_docs --doc $GIT_HOME/enterprise-search-pub
 
 alias docbldews='$GIT_HOME/docs/build_docs --doc $GIT_HOME/enterprise-search-pubs/workplace-search-docs/index.asciidoc --chunk=1'
 
-alias docbldeesp='$GIT_HOME/docs/build_docs --doc $GIT_HOME/enterprise-search-php/docs/guide/index.asciidoc'
```

Comment:
Drive by change. Renamed this alias because it was duplicating the Python alias.